backend/mo/canisters/ic_vetkeys_manager_canister/src/Main.mo

```diff
@@ -0,0 +1,81 @@
+import { KeyManager } "../../../ic_vetkeys/src";
+import Types "../../../ic_vetkeys/src/Types";
+import Principal "mo:base/Principal";
+import Text "mo:base/Text";
+import Blob "mo:base/Blob";
+import Result "mo:base/Result";
+
+actor {
+    var keyManager = KeyManager.KeyManager<Types.AccessRights>("key_manager", Types.accessRightsOperations());
+    public type ByteBuf = { inner : Blob };
+    public type Result<Ok, Err> = {
+        #Ok : Ok;
+        #Err : Err;
+    };
+
+    public query (msg) func get_accessible_shared_key_ids() : async [(Principal, Blob)] {
+        keyManager.getAccessibleSharedKeyIds(msg.caller);
+    };
+
+    public query (msg) func get_shared_user_access_for_key(
+        key_owner : Principal,
+        key_name : Blob,
+    ) : async Result<[(Principal, Types.AccessRights)], Text> {
+        convertResult(keyManager.getSharedUserAccessForKey(msg.caller, (key_owner, key_name)));
+    };
+
+    public shared func get_vetkey_verification_key() : async ByteBuf {
```

Comment:
Here, both the Rust canister and the Motoko canister return a ByteBuf, so the same type is used, while for other methods the Motoko canister sometimes specifies `Blob` instead of `ByteBuf`.